gradio/components/chatbot.py

```diff
@@ -181,8 +168,9 @@ def __init__(
         likeable: bool = False,
         layout: Literal["panel", "bubble"] | None = None,
         placeholder: str | None = None,
-        suggestions: list[SuggestionMessage] | None = None,
         show_copy_all_button=False,
+        examples: list[str] | None=None,
+        show_examples_inside: bool = True,
```

Comment:
Maybe rename this to `show_examples_in_chatbot`?

Reply:
Actually, why do we even need this parameter? No other component supports examples like this. If a user is passing in examples using the `examples` parameter of `Chatbot`, can't we assume they want examples inside the chatbot?

Reply:
hmm a bit confusing, but if you pass examples= directly, then they automatically render inside but don't do anything when clicked other than trigger a `load_example` listener. show_examples_inside=True means that any gr.Examples that outputs to this chatbot automatically is automatically rendered in the chatbot instead.